### PR TITLE
updated MODEL-REALITY-DIVERGENCE

### PR DESCRIPTION
**Pull request by ipa-hsd**
As per https://github.com/ChrisTimperley/ese-robust/issues/30#issuecomment-876678489, `SYSTEM:MODEL-REALITY-DIVERGENCE` labels are updated to `SYSTEM:BEHAVING-INCORRECTLY` and/or `SYSTEM:MONITORING`.

The PR is stacked behind https://github.com/robust-rosin/robust/pull/437

### CODE REVIEW

File: geometry2/860b866/860b866.bug [hunk 1]
```diff
@@ -86,4 +86,4 @@ fault-codes:
 failure-codes:
   - SOFTWARE:BEHAVIORAL
   - SYSTEM:BEHAVING-INCORRECTLY
-  - SYSTEM:MODEL-REALITY-DIVERGENCE
+  - SYSTEM:MONITORING
```
Comment (ChrisTimperley) [inline geometry2/860b866/860b866.bug:89]: I don't quite see the case for MONITORING on this one. This bug leads to a bad vector translation in geometry2, which fits best with our BEHAVING-INCORRECTLY label. I suggest that we drop the MONITORING label.

File: geometry2/d50d51b/d50d51b.bug [hunk 2]
```diff
@@ -60,4 +60,4 @@ fault-codes:
   - PROGRAMMING:VALIDATION
 failure-codes:
   - SOFTWARE:BEHAVIORAL
-  - SYSTEM:MODEL-REALITY-DIVERGENCE
+  - SYSTEM:MONITORING
```
Comment (ChrisTimperley) [inline geometry2/d50d51b/d50d51b.bug:63]: I don't see the fit for monitoring here, either. The bug seems to be that static transforms aren't published with timestamps or frame identifiers. BEHAVING-INCORRECTLY feels more appropriate?

File: kobuki/2f647af/2f647af.bug [hunk 3]
```diff
@@ -151,4 +151,4 @@ fault-codes:
   - PROGRAMMING:CALCULATIONS
 failure-codes:
   - SOFTWARE:BEHAVIORAL
-  - SYSTEM:MODEL-REALITY-DIVERGENCE
+  - SYSTEM:MONITORING
```
Comment (ChrisTimperley) [inline kobuki/2f647af/2f647af.bug:154]: This bug also seems like a clear case of BEHAVING-INCORRECTLY to me due to incorrect odometry information.

File: mavros/753226d/753226d.bug [hunk 4]
```diff
@@ -74,4 +74,4 @@ fault-codes:
   - CONCURRENCY:TIMING
 failure-codes:
   - SOFTWARE:BEHAVIORAL
-  - SYSTEM:MODEL-REALITY-DIVERGENCE
+  - SYSTEM:MONITORING
```
Comment (ChrisTimperley) [inline mavros/753226d/753226d.bug:77]: Seems like another case of BEHAVING-INCORRECTLY: "Therefore, any subscribers
  which rely on this timing information will behave incorrectly."

File: mavros/fcf9cd9/fcf9cd9.bug [hunk 5]
```diff
@@ -70,4 +70,4 @@ fault-codes:
   - PROGRAMMING:UNINITIALISED
 failure-codes:
   - SOFTWARE:BEHAVIORAL
-  - SYSTEM:MODEL-REALITY-DIVERGENCE
+  - SYSTEM:MONITORING
```
Comment (ChrisTimperley) [inline mavros/fcf9cd9/fcf9cd9.bug:73]: Incorrect timestamps seems like a good fit for BEHAVING-INCORRECTLY to me. MONITORING could also fit here too?

File: other/22e4e4f/22e4e4f.bug [hunk 6]
```diff
@@ -47,5 +47,5 @@ fault-codes:
   - PROGRAMMING:BROKEN-CONTRACT
 failure-codes:
   - SOFTWARE:BEHAVIORAL
-  - SYSTEM:MODEL-REALITY-DIVERGENCE
+  - SYSTEM:MONITORING
```
Comment (ChrisTimperley) [inline other/22e4e4f/22e4e4f.bug:50]: What's the reason for MONITORING on this one?

File: kobuki/2f647af/2f647af.bug (hunk 3 above)
Comment (ipa-hsd) [reply kobuki/2f647af/2f647af.bug:154]: My rationale was that messages like odometry and joint states would be used to calculate the inner state of the robot (eventually they might be used to calculate control commands and lead to incorrect behaviour - but we considered such cases as extrapolation before)

**Review by ChrisTimperley — CHANGES_REQUESTED** [review]
I think that we'll want to discuss and possibly change a few labels at today's meeting.

File: other/22e4e4f/22e4e4f.bug (hunk 6 above)
Comment (ipa-hsd) [reply other/22e4e4f/22e4e4f.bug:50]: same as https://github.com/robust-rosin/robust/pull/441#discussion_r682619842

File: geometry2/860b866/860b866.bug (hunk 1 above)
Comment (ChrisTimperley) [reply geometry2/860b866/860b866.bug:89]: From the meeting: Let's keep both labels. MONITORING failures would be observed by the user when using, e.g., rviz. BEHAVING-INCORRECTLY would occur in all places where this transformation is used.

File: geometry2/d50d51b/d50d51b.bug (hunk 2 above)
Comment (ChrisTimperley) [reply geometry2/d50d51b/d50d51b.bug:63]: From the meeting: let's do both MONITORING (due to frame identifiers) and BEHAVING-INCORRECTLY.

File: kobuki/2f647af/2f647af.bug (hunk 3 above)
Comment (ChrisTimperley) [reply kobuki/2f647af/2f647af.bug:154]: From the meeting: let's go with both BEHAVING-INCORRECTLY and MONITORING.

File: mavros/fcf9cd9/fcf9cd9.bug (hunk 5 above)
Comment (ChrisTimperley) [reply mavros/fcf9cd9/fcf9cd9.bug:73]: Same reasons as above: Both BEHAVING-INCORRECTLY and MONITORING apply. (The operator may indirectly rely on information that is derived from timestamps [e.g., the UAV's orientation]).

File: other/22e4e4f/22e4e4f.bug (hunk 6 above)
Comment (ChrisTimperley) [reply other/22e4e4f/22e4e4f.bug:50]: I have no idea why I put this comment here. This is an obvious MONITORING issue :facepalm:

**Comment by ChrisTimperley** [comment]
@ipa-hsd Any updates on the requested changes?

**Comment by ipa-hsd** [comment]
@ChrisTimperley I have updated the labels

**Review by ChrisTimperley — APPROVED** [review]

**Pull request merged**